roqoqo-qryd/tests/integration/tweezer_devices.rs

```diff
@@ -587,22 +591,30 @@ fn test_from_api() {
                 .unwrap()
                 .into_bytes(),
         )
+        .expect(2)
```

Comment:
The mock expects two hits, the first one coming from a normal `TweezerDevice::from_api()` call (line 597) and the second one from the added call with a specific seed value (line 604). Since the response is the same (the device seed is changed after it is retrieved), I simply modified the already existing mock.